### PR TITLE
SoF sound track

### PR DESCRIPTION
**Pull request by doofus-01**
I guess this is subjective, but the sound track to this campaign seemed very ... default.

### CODE REVIEW

File: data/campaigns/Sceptre_of_Fire/scenarios/2_Closing_the_Gates.cfg [hunk 1]
```diff
@@ -157,6 +158,11 @@
             [/gold_carryover]
         [/objectives]
 
+        [music]
+            name=battle.ogg
+            append=yes
+        [/music]
+
```
Comment (nemaara) [inline data/campaigns/Sceptre_of_Fire/scenarios/2_Closing_the_Gates.cfg:165]: Think adding another track here could be nice.

File: data/campaigns/Sceptre_of_Fire/scenarios/4_Gathering_Materials.cfg [hunk 2]
```diff
@@ -323,6 +324,11 @@
         # wmllint: recognize Alanin
         # wmllint: recognize Krawg
 
+        [music]
+            name=into_the_shadows.ogg
```
Comment (nemaara) [inline data/campaigns/Sceptre_of_Fire/scenarios/4_Gathering_Materials.cfg:328]: Somehow I feel like into the shadows doesn't fit well here. If you don't like suspense, then maybe nunc dimittis?

File: data/campaigns/Sceptre_of_Fire/scenarios/2_Closing_the_Gates.cfg (hunk 1 above)
Comment (doofus-01) [reply data/campaigns/Sceptre_of_Fire/scenarios/2_Closing_the_Gates.cfg:165]: Sure!  Will add.

File: data/campaigns/Sceptre_of_Fire/scenarios/4_Gathering_Materials.cfg (hunk 2 above)
Comment (doofus-01) [reply data/campaigns/Sceptre_of_Fire/scenarios/4_Gathering_Materials.cfg:328]: I'll give that a try, thanks.

**Comment by doofus-01** [comment]
There are specific choices I'd like to change, but my main issue with the sound track is the "revelation.ogg" during intros.

**Comment by Pentarctagon** [comment]
@doofus-01 is this something you were still intending to do?

**Comment by doofus-01** [comment]
Idealistically, yes.  Realistically, no, not anytime soon.

**Comment by Pentarctagon** [comment]
Alright, I'm going to close it for now then given how long it's been open already. Feel free to re-open it if/when you're able to work on it again though.

**Pull request closed without merging**